proto/features.proto

```diff
@@ -16,7 +16,9 @@ enum FeatureType {
     FeatureTypePath = 1;
     FeatureTypeArea = 2;
     FeatureTypeRelation = 3;
-    FeatureTypeInvalid = 4;
+    FeatureTypeCollection = 4;
+    FeatureTypeExpression = 5;
+    FeatureTypeInvalid = 6;
```

Comment:
ye like here for invalid, i mean u do u, but we both know what the right thing to do is !!